packages/dashboard/src/app/router/route.js

```diff
@@ -67,22 +67,58 @@ export function resolveRoute(route) {
   }
 }
 
-export function matchPath(currentPath, path, exact = false) {
+export function matchPath({
+  currentPath,
+  path,
+  availableRoutes,
+  defaultRoute,
+  isDefault = false,
+  exact = false,
+}) {
   const match = new RegExp(`^${path}`).exec(currentPath);
-  if (!match) {
+
+  // allow paths through that don't match if they are default, not narrowed out yet.
+  if (!match && !isDefault) {
```

Comment:
yup! so `stories-dashboard#wpbody-content` comes back as `match = null` so it will come back `true` for `!match` - the thing is, that I want to find out if there's no match _and_ if it's default we want to check and see if the currentPath starts with one of our available routes. It means that there's no chance that this currentPath would be represented by another available path that would render instead.